custom_components/tuya_local/devices/doel_tiplus_litterbox.yaml

```diff
@@ -0,0 +1,249 @@
+name: Litter Box
```

Comment:
HA Naming convention: only the first word should be capitalised in device and entity names (applies to quite a few entity names below too)

https://developers.home-assistant.io/blog/2022/07/10/entity_naming/